### ISSUE TITLE
Perfect Vision Compatibility

### ISSUE THREAD

**Issue by ravingdragoon** [ggagnon76/token-independence#30]
When running in Foundry V9 with Perfect Vision I receive the following error. I have also reported the error with the Perfect Vision Git.
```
TypeError: Cannot read properties of undefined (reading 'ActorName')
[Detected 2 packages: token-independence, perfect-vision]
    at populateSynthetics (helper.js:53)
    at independence.js:9
    at Function._call (foundry.js:294)
    at Function.call (foundry.js:279)
    at Canvas.draw (foundry.js:19879)
    at async Canvas.🎁Canvas.prototype.draw#perfect-vision (canvas.js:345)
    at async Scene.view (foundry.js:17642)
    at async Game.initializeCanvas (foundry.js:5615)
    at async Game.setupGame (foundry.js:5484)
    at async Game._initializeGameView (foundry.js:6434)
    at async Game.initialize (foundry.js:5380)
onError	@	foundry.js:316
🎁call_wrapped	@	libWrapper-wrapper.js:502
🎁Hooks.onError#lib-wrapper	@	listeners.js:137
🎁Hooks.onError#0	@	libWrapper-wrapper.js:182
_call	@	foundry.js:298
call	@	foundry.js:279
draw	@	foundry.js:19879
```

**Comment by ggagnon76** [comment]
Thank you.  I will look into it.

What version of Perfect Vision do you have installed?

**Comment by ggagnon76** [comment]
Actually, I don't think it has anything to do with Perfect Vision.  I suspect you had a broken token pre-existing in the scene and my module is trying to find a flag that isn't there.  I believe it will be an easy fix.

**Comment by ravingdragoon** [comment]
Apologies, you are correct, the error still occurs with PV disabled.

**Comment by ggagnon76** [comment]
Should be resolved with version 0.2.1
